### PR TITLE
Feature/Resend Email page

### PR DESCRIPTION
**Pull request by alexkocharyan01**

### CODE REVIEW

File: src/pages/resend/index.tsx [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+            leftIcon={<LeftIcon />}
+          >
+            Go back
+          </Button>
+        </Link>
+        <Link href="/login">
+          <Button variant="outlined">Log In</Button>
+        </Link>
+      </div>
+      <div className="mt-[60] xl:mt-10">
```
Comment (rikarsen) [inline src/pages/resend/index.tsx:23]: maybe mt-60?

Comment (rikarsen) [reply src/pages/resend/index.tsx:23]: font sized and weights different in mobile and desktop sizes, please be careful

File: src/pages/resend/index.tsx [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+        <h1 className="text-xl text-black">The link was sent to your email</h1>
+        <div className="mt-5 flex items-center border-b border-gray-thin pb-5">
+          <svg
```
Comment (rikarsen) [inline src/pages/resend/index.tsx:26]: create custom svg component like we did in other pages

File: src/pages/resend/index.tsx [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+              fill="#2FB32C"
+            />
+          </svg>
+          <p className="text-base text-black-light">
+            Check your inbox and click on the link for password recovery
+          </p>
+        </div>
+        <div className="mt-5">
+          <p className="text-base text-black-light">Didn’t get the password? Click “Resend”</p>
+          <Button className="mt-5 w-full">Resend Email</Button>
```
Comment (rikarsen) [inline src/pages/resend/index.tsx:45]: use button props for button styles instead of className

File: src/pages/resend/index.tsx [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+            Go back
+          </Button>
+        </Link>
+        <Link href="/login">
+          <Button variant="outlined">Log In</Button>
+        </Link>
+      </div>
+      <div className="mt-[60] xl:mt-10">
+        <h1 className="text-xl text-black">The link was sent to your email</h1>
+        <div className="mt-5 flex items-center border-b border-gray-thin pb-5">
```
Comment (rikarsen) [inline src/pages/resend/index.tsx:25]: items-center is unnecessary

**Review by rikarsen — CHANGES_REQUESTED** [review]

File: src/pages/reset-password/index.tsx [hunk 5]
```diff
@@ -22,24 +43,102 @@ export const ResetPassword = () => {
...
...1.7097 25.5739 21.7729 25.418 21.8257L23.5441 22.4632C23.0357 22.6363 22.6363 23.0357 22.4632 23.5441L21.8257 25.418C21.1487 27.4085 18.9862 28.4732 16.9956 27.7962C16.8399 27.7431 16.6877 27.6801 16.5402 27.6075L14.7643 26.7331C14.2824 26.4958 13.7176 26.4958 13.2357 26.7331L11.4599 27.6075C9.57361 28.5362 7.2916 27.7599 6.36289 25.8736C6.29021 25.726 6.22718 25.5739 6.17421 25.418L5.53677 23.5441C5.36377 23.0357 4.96438 22.6363 4.45581 22.4632L2.58196 21.8257C0.591448 21.1487 -0.473254 18.9862 0.203858 16.9956C0.256851 16.8399 0.319881 16.6877 0.392545 16.5402L1.26683 14.7643C1.50411 14.2824 1.50411 13.7176 1.26683 13.2357L0.392545 11.4599C-0.536159 9.57361 0.240129 7.2916 2.12641 6.36289C2.27403 6.29021 2.42619 6.22718 2.58196 6.17421L4.45581 5.53677C4.96438 5.36377 5.36377 4.96438 5.53677 4.45581L6.17421 2.58196C6.85132 0.591448 9.01386 -0.473254 11.0044 0.203858ZM18.8033 9.80498L11.3011 17.3071L8.56805 14.0276C8.20096 13.5869 7.54627 13.5274 7.10576 13.8945C6.66525 14.2617 6.60573 14.9163 6.97282 15.3568L10.4337 19.5099C10.8245 19.9789 11.5338 20.0111 11.9654 19.5794L20.2715 11.2733C20.677 10.8678 20.677 10.2104 20.2715 9.80498C19.8661 9.39951 19.2088 9.39951 18.8033 9.80498Z"
```
Comment (ArthurO2002) [inline src/pages/reset-password/index.tsx:89]: Move the svg to another file

File: src/pages/reset-password/index.tsx [hunk 6]
```diff
@@ -1,14 +1,35 @@
...
+  const [emailValue, setEmailValue] = useState<string>('')
+
+  const handleSend = () => {
+    const emailVerification: boolean = /\S+@\S+\.\S+/.test(emailValue)
+
+    if (!emailVerification || !emailValue.length) {
+      return
+    }
+
+    router.query.tab = 'verify-otp'
```
Comment (ArthurO2002) [inline src/pages/reset-password/index.tsx:20]: should be enum

File: src/pages/reset-password/index.tsx [hunk 7]
```diff
@@ -1,14 +1,35 @@
...
   const error = false
+  const router = useRouter()
+  const [emailValue, setEmailValue] = useState<string>('')
+
+  const handleSend = () => {
+    const emailVerification: boolean = /\S+@\S+\.\S+/.test(emailValue)
+
+    if (!emailVerification || !emailValue.length) {
+      return
+    }
```
Comment (ArthurO2002) [inline src/pages/reset-password/index.tsx:18]: in case of error let's put error message

**Review by ArthurO2002 — CHANGES_REQUESTED** [review]

File: src/pages/reset-password/index.tsx (hunk 7 above)
Comment (alexkocharyan01) [reply src/pages/reset-password/index.tsx:18]: dane

File: src/pages/reset-password/index.tsx (hunk 5 above)
Comment (alexkocharyan01) [reply src/pages/reset-password/index.tsx:89]: done

File: src/pages/resend/index.tsx [hunk 8]
```diff
@@ -0,0 +1,52 @@
+import React from 'react'
```
[the remark this answers is not in the document]
Comment (alexkocharyan01) [reply src/pages/resend/index.tsx]: done

File: src/pages/resend/index.tsx (hunk 1 above)
Comment (alexkocharyan01) [reply src/pages/resend/index.tsx]: done

File: src/pages/resend/index.tsx (hunk 2 above)
Comment (alexkocharyan01) [reply src/pages/resend/index.tsx]: done

File: src/pages/resend/index.tsx (hunk 3 above)
Comment (alexkocharyan01) [reply src/pages/resend/index.tsx]: done

File: src/pages/resend/index.tsx (hunk 4 above)
Comment (alexkocharyan01) [reply src/pages/resend/index.tsx]: done

File: src/pages/reset-password/index.tsx [hunk 9]
```diff
@@ -1,14 +1,45 @@
...
+  const handleEmailSend = () => {
+    const emailVerification: boolean = /\S+@\S+\.\S+/.test(emailValue)
+
+    if (!emailVerification || !emailValue.length) {
+      setEmailVerificationError('Email not valid')
+
+      return
+    }
+
+    router.query.tab = 'verify-otp'
```
Comment (ArthurO2002) [inline src/pages/reset-password/index.tsx:29]:
```suggestion
    router.query.tab = RouterQueryTypes.verifyOtp
```

File: src/pages/reset-password/index.tsx [hunk 10]
```diff
@@ -22,24 +53,72 @@ export const ResetPassword = () => {
...
+              <Input
+                type="email"
+                error={emailVerificationError}
+                placeholder="Email"
+                onChange={(e) => setEmailValue(e.target.value)}
+              />
+            </div>
+            <Button
+              size="fl"
+              onClick={() => handleEmailSend()}
```
Comment (ArthurO2002) [inline src/pages/reset-password/index.tsx:74]:
```suggestion
              onClick={handleEmailSend}
```

File: src/pages/reset-password/index.tsx (hunk 9 above)
Comment (alexkocharyan01) [reply src/pages/reset-password/index.tsx:29]: done

File: src/pages/reset-password/index.tsx (hunk 10 above)
Comment (alexkocharyan01) [reply src/pages/reset-password/index.tsx:74]: done

File: src/pages/reset-password/index.tsx [hunk 11]
```diff
@@ -22,24 +53,72 @@ export const ResetPassword = () => {
...
+              </p>
+            </div>
+            <div className="mb-5 mt-5">
+              <p className="mb-5 text-base text-black-light">
+                Didn’t get the password? Click “Resend”
+              </p>
+              <Input
+                type="email"
+                placeholder="Enter Code"
+                onChange={(e) => handleEnterCode(e.target.value)}
```
[the remark this answers is not in the document]
Comment (alexkocharyan01) [reply src/pages/reset-password/index.tsx:98]: done

File: src/pages/reset-password/index.tsx [hunk 12]
```diff
@@ -22,24 +53,72 @@ export const ResetPassword = () => {
...
+            <p className="mb-5 text-base text-black-light">
+              Enter the email that you’ve registered with in the field below. You will receive a
+              link for password recovery
+            </p>
+            <div className="mb-5 w-full">
+              <Input
+                type="email"
+                error={emailVerificationError}
+                placeholder="Email"
+                onChange={(e) => setEmailValue(e.target.value)}
```
[the remark this answers is not in the document]
Comment (alexkocharyan01) [reply src/pages/reset-password/index.tsx:69]: done

File: src/pages/reset-password/index.tsx (hunk 6 above)
Comment (alexkocharyan01) [reply src/pages/reset-password/index.tsx]: done

File: src/pages/reset-password/index.tsx [hunk 13]
```diff
@@ -22,24 +57,72 @@ export const ResetPassword = () => {
           <Button variant="outlined">Log In</Button>
```
Comment (rikarsen) [inline src/pages/reset-password/index.tsx:57]: show register button instead of login

File: src/components/UI-Components/Input/Input.tsx [hunk 14]
```diff
@@ -0,0 +1,38 @@
+import React, { ChangeEvent, FC } from 'react'
```
Comment (rikarsen) [inline src/components/UI-Components/Input/Input.tsx:1]: move to right place

File: src/components/UI-Components/Input/Input.tsx [hunk 15]
```diff
@@ -0,0 +1,38 @@
...
+    'rounded',
+    'px-5',
+    'py-2.5',
+    'outline-0',
+    'placeholder:text-base',
+    'border-gray-light',
+    'placeholder:text-black',
+    'w-full',
+    [error ? 'border-red placeholder:text-red' : 'border-gray-light placeholder:text-black']
+  )
```
Comment (rikarsen) [inline src/components/UI-Components/Input/Input.tsx:24]: please update input styles to fit with design

File: src/components/UI-Components/Input/Input.tsx [hunk 16]
```diff
@@ -0,0 +1,38 @@
...
+  return (
+    <>
+      <input
+        type={type}
+        placeholder={placeholder}
+        id={id}
+        className={style}
+        onChange={onChange}
+      />
+      {error ? <p className="mt-2.5 text-red">{error}</p> : null}
```
Comment (rikarsen) [inline src/components/UI-Components/Input/Input.tsx:35]: update design

File: src/pages/reset-password/index.tsx [hunk 17]
```diff
@@ -22,24 +57,72 @@ export const ResetPassword = () => {
...
-          />
-          {error ? <p className="mt-2.5 text-red">Email not valid</p> : null}
-        </div>
-        <Button size="fl">Next</Button>
+      <div className="flex w-full justify-center">
+        {!router.query.tab && (
+          <div className="w-full max-w-[480px]">
+            <h1 className="mb-5 text-xl">Forgot your password</h1>
+            <p className="mb-5 text-base text-black-light">
+              Enter the email that you’ve registered with in the field below. You will receive a
```
Comment (rikarsen) [inline src/pages/reset-password/index.tsx:65]: update text

File: src/components/UI-Components/Input/Input.tsx (hunk 16 above)
Comment (rikarsen) [reply src/components/UI-Components/Input/Input.tsx:35]: add eye icon for type="password"

File: src/components/Icons/CheckInIcon.tsx [hunk 18]
```diff
@@ -0,0 +1,17 @@
+import React from 'react'
+
+export const ChekInIcon = () => (
+  <svg
+    width="28"
+    height="28"
+    viewBox="0 0 28 28"
+    fill="none"
+    xmlns="http://www.w3.org/2000/svg"
+    className="mr-5"
```
Comment (rikarsen) [inline src/components/Icons/CheckInIcon.tsx:10]: provide className with props

**Review by rikarsen — CHANGES_REQUESTED** [review]

File: src/pages/reset-password/index.tsx (hunk 13 above)
Comment (alexkocharyan01) [reply src/pages/reset-password/index.tsx:57]: done

File: src/components/UI-Components/Input/Input.tsx (hunk 14 above)
Comment (alexkocharyan01) [reply src/components/UI-Components/Input/Input.tsx:1]: done

File: src/components/UI-Components/Input/Input.tsx (hunk 15 above)
Comment (alexkocharyan01) [reply src/components/UI-Components/Input/Input.tsx:24]: done

File: src/pages/reset-password/index.tsx (hunk 17 above)
Comment (alexkocharyan01) [reply src/pages/reset-password/index.tsx:65]: done

File: src/components/Icons/CheckInIcon.tsx (hunk 18 above)
Comment (alexkocharyan01) [reply src/components/Icons/CheckInIcon.tsx:10]: done

File: src/components/UI-Components/Input/Input.tsx (hunk 16 above)
Comment (alexkocharyan01) [reply src/components/UI-Components/Input/Input.tsx:35]: done

**Review by ArthurO2002 — APPROVED** [review]

**Review by rikarsen — APPROVED** [review]

**Pull request merged**